ios/ReactNativeHealthkit.swift

```diff
@@ -1034,7 +1035,7 @@ class ReactNativeHealthkit: RCTEventEmitter {
 
         let q = HKCorrelationQuery(type: sampleType, predicate: predicate, samplePredicates: nil) { (query: HKCorrelationQuery, _correlations: [HKCorrelation]?, error: Error?) in
             guard let err = error else {
-                guard let correlations = _correlations else {
+                guard let correlations = _correlations, !correlations.isEmpty  else {
```

Comment:
Not quite sure if this makes a difference? I'm guessing this catches empty arrays (not a Swift expert), but that should be handle by the guarded case anyways, right?